src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,21 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.service.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    public static void main(String[] args) throws RegistrationException, AuthenticationException {
+        MovieService movieService = (MovieService) Injector.getInstance("mate")
+                .getInstance(MovieService.class);
```

Comment:
```suggestion
        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
```
create a field in class for the injector